locale broken after update

<!-- IMPORTANT: Please disable plugins prior to posting a bug report.  If you have a problem with a plugin please post on the plugin repository.  Thanks! -->

**Describe the bug**
When I update from v2.1.0 to v2.2.5, the language option seems broken. I can't choose some languages like Chinese.

**To Reproduce**
Steps to reproduce the behavior:
1. Stop the instance
2. Do a MySQL backup of the database
3. Do `git pull` on `master` branch
4. Do `pnpm i`
5. Do `pnpm run build:etherpad`
6. Do `pnpm run plugins i ep_align ep_comments_page ep_embedded_hyperlinks2 ep_font_color ep_guest ep_headings2 ep_markdown ep_openid_connect ep_stable_authorid ep_user_displayname ep_webrtc` to update my plugins
7. Do `pnpm run prod`

**Expected behavior**
It should have shown Chinese characters, but it didn't. Also, I can't select Chinese in the language option.

**Screenshots**

No characters on the main page.

![image](https://github.com/user-attachments/assets/ab605b07-991a-4bef-b5e8-4e41c4aaed4b)

When I choose Chinese (中文 中国大陆), it won't change to Chinese and shows blank.

![image](https://github.com/user-attachments/assets/e5ad187f-3445-4fd2-9bdb-6a8d13cbf750)

**Server (please complete the following information):**
 - Etherpad version: 2.2.5
 - OS: Ubuntu 24.04.1 LTS
 - Node.js version (`node --version`): v20.18.0
 - npm version (`npm --version`): 10.8.2
 - Is the server free of plugins: No. Here are the plugins I installed:

```
  ep_align
  ep_comments_page
  ep_embedded_hyperlinks2
  ep_font_color
  ep_guest
  ep_headings2
  ep_markdown
  ep_openid_connect
  ep_stable_authorid
  ep_user_displayname
  ep_webrtc
```

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOS
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 129.0.6668.101

**Additional context**
There's seems nothing useful in the logs:

```
[2024-10-24T08:29:37.786] [INFO] settings - You can access your Etherpad instance at http://0.0.0.0:9001/
[2024-10-24T08:29:37.787] [INFO] settings - The plugin admin page is at http://0.0.0.0:9001/admin/plugins
[2024-10-24T08:29:37.787] [INFO] server - Etherpad is running
[2024-10-24T08:29:40.380] [INFO] access - [ENTER] pad:TechSalon socket:gOXzskhBCulAAkD-AAAF IP:xxx.xxx.xxx.xxx authorID:a.lHOwncdCbYCoQ8N3 username:88
[2024-10-24T08:29:54.466] [INFO] http - Successful authentication from IP xxx.xxx.xxx.xxx for user guest
[2024-10-24T08:29:54.563] [INFO] Minify - Compress CSS file skins/colibris/index.css.
[2024-10-24T08:29:54.575] [INFO] Minify - Compress JS file skins/colibris/index.js.
[2024-10-24T08:30:02.430] [INFO] Minify - Compress CSS file css/pad.css.
[2024-10-24T08:30:02.431] [INFO] Minify - Compress CSS file skins/colibris/pad.css.
[2024-10-24T08:30:02.432] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_webrtc/static/css/styles.css.
[2024-10-24T08:30:02.432] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_comments_page/static/css/main.css.
[2024-10-24T08:30:02.433] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_comments_page/static/css/comment.css.
[2024-10-24T08:30:02.686] [INFO] Minify - Compress JS file skins/colibris/pad.js.
[2024-10-24T08:30:02.735] [INFO] Minify - Compress JS file /opt/etherpad/src/plugin_packages/ep_comments_page/static/js/jquery.tmpl.min.js.
[2024-10-24T08:30:03.433] [INFO] access - [ENTER] pad:xxx socket:YqQ3VdmkNSnH0VcgAAAH IP:xxx.xxx.xxx.xxx authorID:a.d4V5GvUgpLfc0nKC username:guest
[2024-10-24T08:30:03.619] [ERROR] socket.io - Error handling pad message from YqQ3VdmkNSnH0VcgAAAH: Error: COLLABROOM: write attempt on read-only pad
    at Object.exports.handleMessage (/opt/etherpad/src/node/handler/PadMessageHandler.ts:379:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async <anonymous> (/opt/etherpad/src/node/handler/SocketIORouter.ts:84:14)
[2024-10-24T08:30:03.628] [INFO] Minify - Compress CSS file css/iframe_editor.css.
[2024-10-24T08:30:03.629] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_markdown/static/css/markdown.css.
[2024-10-24T08:30:03.629] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_headings2/static/css/editor.css.
[2024-10-24T08:30:03.630] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_font_color/static/css/color.css.
[2024-10-24T08:30:03.630] [INFO] Minify - Compress CSS file /opt/etherpad/src/plugin_packages/ep_comments_page/static/css/commentIcon.css.
[2024-10-24T08:30:13.952] [ERROR] socket.io - Error handling pad message from YqQ3VdmkNSnH0VcgAAAH: TypeError: Cannot read properties of undefined (reading 'send')
    at handleRTCMessage (/opt/etherpad/src/plugin_packages/.versions/ep_webrtc@2.3.18/index.js:101:26)
    at exports.handleMessage [as hook_fn] (/opt/etherpad/src/plugin_packages/.versions/ep_webrtc@2.3.18/index.js:231:5)
    at <anonymous> (/opt/etherpad/src/static/js/pluginfw/hooks.ts:273:18)
    at new Promise (<anonymous>)
    at callHookFnAsync (/opt/etherpad/src/static/js/pluginfw/hooks.ts:236:16)
    at <anonymous> (/opt/etherpad/src/static/js/pluginfw/hooks.ts:351:54)
    at Array.map (<anonymous>)
    at Object.exports.aCallAll (/opt/etherpad/src/static/js/pluginfw/hooks.ts:351:13)
    at Object.exports.handleMessage (/opt/etherpad/src/node/handler/PadMessageHandler.ts:332:20)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async <anonymous> (/opt/etherpad/src/node/handler/SocketIORouter.ts:84:14)
[2024-10-24T08:30:13.962] [INFO] access - [LEAVE] pad:xxx socket:YqQ3VdmkNSnH0VcgAAAH IP:xxx.xxx.xxx.xxx authorID:a.d4V5GvUgpLfc0nKC username:guest
[2024-10-24T08:30:14.351] [INFO] access - [ENTER] pad:xxx socket:UuigUObLB2M88Q14AAAL IP:xxx.xxx.xxx.xxx authorID:a.d4V5GvUgpLfc0nKC username:guest
[2024-10-24T08:30:14.554] [ERROR] socket.io - Error handling pad message from UuigUObLB2M88Q14AAAL: Error: COLLABROOM: write attempt on read-only pad
    at Object.exports.handleMessage (/opt/etherpad/src/node/handler/PadMessageHandler.ts:379:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async <anonymous> (/opt/etherpad/src/node/handler/SocketIORouter.ts:84:14)
```

--- a/src/static/js/vendors/html10n.ts
+++ b/src/static/js/vendors/html10n.ts
@@ -538,7 +538,7 @@ export class Html10n {
       for (let i=0, n=langs.length; i < n; i++) {
         lang = langs[i]
         if(!lang) continue;
-        if(!(lang in langs)) {// uh, we don't have this lang availbable..
+        if(!langs.includes(lang)) {// uh, we don't have this lang availbable..
           // then check for related langs
           if(~lang.indexOf('-') != -1) {
             lang = lang.split('-')[0];

